homeassistant/components/modbus/validators.py

```diff
@@ -65,7 +65,7 @@ def struct_validator(config: dict[str, Any]) -> dict[str, Any]:
     name = config[CONF_NAME]
     structure = config.get(CONF_STRUCTURE)
     slave_count = config.get(CONF_SLAVE_COUNT, 0) + 1
-    slave = config.get(CONF_SLAVE, 0)
+    config.get(CONF_SLAVE, 0)
```

Comment:
I don't understand this change, does this really do anything?

Reply:
You are quite right, seems my editor have a life of its own (I am on my ipad away from my mac). Will add a commit in a moment.